moveit_core/kinematics_base/src/kinematics_base.cpp

```diff
@@ -112,17 +110,18 @@ bool KinematicsBase::initialize(const std::string& robot_description, const std:
     return initialize(robot_description, group_name, base_frame, tip_frames[0], search_discretization);
   }
 
-  ROS_ERROR_NAMED(LOGNAME, "This solver does not support multiple tip frames");
+  RCLCPP_ERROR(kinematics::LOGGER, "This solver does not support multiple tip frames");
```

Comment:
```suggestion
  RCLCPP_ERROR(LOGGER, "This solver does not support multiple tip frames");
```
We're already inside the `kinematics` namespace